source/include/FreeRTOS_IP_Private.h

```diff
@@ -258,8 +258,6 @@ struct xPacketSummary
     uint16_t usProtocolBytes;              /**< The total length of the protocol data. */
 };
 
-#define ipBROADCAST_IP_ADDRESS               0xffffffffU
```

Comment:
There are few more references to `ipBROADCAST_IP_ADDRESS` in the code - STM32 network interfaces and UTs